circle.yml

```diff
@@ -1,11 +1,15 @@
 machine:
-  services:
-    - docker
+  pre:
+    - sudo docker -d -e lxc -s btrfs -H tcp://0.0.0.0:2375:
+        background: true
+    - sleep 5
   environment:
     REPO: $CIRCLE_PROJECT_USERNAME/$CIRCLE_PROJECT_REPONAME
+    DOCKER_HOST: tcp://172.17.42.1:2375
 
 dependencies:
   pre:
+    - docker version
```

Comment:
Making sure we can talk to docker at this point (fail early, fail often :stuck_out_tongue_closed_eyes:) and also might be a good way to know if circleci updates their docker version :wink: